script/reference-device/dhcp6_pd_daemon.py

```diff
@@ -107,8 +107,17 @@ def connect_to_signal():
 
 
 def check_and_reconnect(dbus_obj):
+    global bus
     if dbus_obj is None:
         connect_to_signal()
+    bus.watch_name_owner(bus_name='io.openthread.BorderRouter.wpan0',
+                         callback=handle_name_owner_changed)
+
+
+def handle_name_owner_changed(new_owner):
+    if not new_owner:
+        logging.warning("Otbr-agent likely reloaded, reconnecting to D-Bus")
```

Comment:
By `reload`, do you mean restart? 'Reloading' may be confused with reloading the config in this context.

Reply:
I agree the message is confusing because it makes reader think about "systemctl reload otbr-agent" -- in fact "Requires=otbr-agent.service" already guarantees pd daemon restarts on otbr-agent.

To clarify, it is triggered by otbr-agent's internal reset mechanism (e.g. during `factoryreset`), which also re-establishes the dbus connection.

I'll update the message to use 'reset' instead of 'reload' to avoid confusion.